packages/@react-native/tester/js/examples/MultiColumn/MultiColumnExample.js

```diff
@@ -151,8 +167,18 @@ class MultiColumnExample extends React.PureComponent<
       );
     }
   };
+
   _pressItem = (key: string) => {
-    pressItem(this, key);
+    const index = Number(key);
+    const itemState = pressItem(this.state.data[index]);
+    this.setState(state => ({
```

Comment:
nit on the fb code: This is duplicate from above, should have a helper function to insert at position 1.